traefik_mesh/README.md

```diff
@@ -40,7 +40,70 @@ Make sure that the Prometheus-formatted metrics are exposed in your Traefik Mesh
 When configuring the Traefik Mesh check, you can use the following parameters:
 - `openmetrics_endpoint`: This parameter should be set to the location where the Prometheus-formatted metrics are exposed. The default port is `8082`, but it can be configured using the `--entryPoints.metrics.address`. In containerized environments, `%%host%%` can be used for [host autodetection][3].
 - `traefik_proxy_api_endpooint:` This parameter is optional. The default port is `8080` and can be configured using `--entryPoints.traefik.address`. In containerized environments, `%%host%%` can be used for [host autodetection][3].
-- `traefik_controller_api_endpoint`: This parameter is optional. The default port is set to `9000`. 
+- `traefik_controller_api_endpoint`: This parameter is optional. The default port is set to `9000`.
+
+#### Traefik Proxy
+```yaml
+# (...)
+metadata:
+  name: '<POD_NAME>'
+  annotations:
+    ad.datadoghq.com/<CONTAINER_NAME>.checks: |
+      {
+        "traefik_mesh": {
+          "init_config": {},
+          "instances": [
+            {
+              "openmetrics_endpoint": "http://%%host%%:8082/metrics",
+              "traefik_proxy_api_endpoint": "http://%%host%%:8080"
```

Comment:
8080 or 9000?

Reply:
Proxy API is 8080; controller API is 9000